openedx/core/lib/block_structure/transformer_registry.py

```diff
@@ -45,8 +45,8 @@ def get_write_version_hash(cls):
 
         sorted_transformers = sorted(cls.get_registered_transformers(), key=lambda t: t.name())
         for transformer in sorted_transformers:
-            hash_obj.update(transformer.name())
-            hash_obj.update(unicode(transformer.WRITE_VERSION))
+            hash_obj.update(transformer.name().encode('utf-8'))
+            hash_obj.update(unicode(transformer.WRITE_VERSION).encode('utf-8'))
```

Comment:
This works, but why incur the extra conversion on L49, rather than just going directly from `int` to `str`?